src/_guides/language/language-tour.md

```diff
@@ -106,22 +106,14 @@ mind:
     (You might notice when you’re debugging your code that
     variables with no specified type get a special type: `dynamic`.)
     Specifying static types is optional in Dart 1.x, however Dart is moving
-    towards being a fully type safe language. See the next bullet
-    for information on strong mode, which is part of that effort.
+    towards being a fully type safe language.
 
 -   _Strong mode_ contributes to making Dart a type safe language&mdash;you'll
```

Comment:
I think we can have fewer words here. How about:

In [strong mode](/guides/language/sound-dart), static and runtime checks ensure that your code is type safe, helping you catch bugs in development, rather than at runtime. Strong mode is optional in Dart 1.x but required in [Dart 2.0](/dart-2.0).

(I'm not sure that "strong mode" is exactly in Dart 2.0. It's more like it's not needed any more in Dart 2.0 because everything's type safe.)